css/index.css

```diff
@@ -143,4 +236,78 @@ footer nav {
 footer nav a {
     color: white;
     text-decoration: none;
-}
\ No newline at end of file
+}
+
+/*-------------------------------------------form----------------------------------------------------*/
+
+.form {
+    background-color: #f2f2f2; 
+    border-radius: 10px;
+    margin: 30px 20px;
+    padding: 25px;
+
+}
+.contact-form {
+    background-color: #f2f2f2; 
+    margin-bottom: 10px;
+    border-radius: 10px;
+}
+
+.contact-form:last-child {
+    margin-bottom: -10px; 
+}
+
+.contact-form label {
+    padding-top: 20px;
+}
+
+.input-label {
+    margin: 6px 0 26px 0; 
+    font-size: 14px;
+    text-align: top;
+    font-weight: bold;
+}
+
+.input-box {
+    margin: 6px 0 26px 0; 
+    font-size: 14px;
+    text-align: top;
+    font-style: oblique;
+}
+
+input[type=text], select, textarea {
```

Comment:
great job using attribute selectors! this is some advanced CSS selectors that we did not cover in class, and you researched it yourself!